src/Nox.Types.Abstractions/TypeOptions/MoneyTypeOptions.cs

```diff
@@ -4,14 +4,14 @@ namespace Nox.Types
 
     public class MoneyTypeOptions : INoxTypeOptions
     {
-        public int DecimalDigits { get; internal set; } = 4;
+        public int DecimalDigits { get; set; } = 4;
```

Comment:
why did we remove the internal, and just for some fields and for some type?

Reply:
Updated.